plasmapy/physics/exceptions.py

```diff
@@ -0,0 +1,44 @@
+"""Exceptions and warnings specific to PlasmaPy."""
```

Comment:
Maybe change "specific to PlasmaPy" to "related to plasma parameter calculations."